packages/reef-knot/package.json

```diff
@@ -15,6 +15,8 @@
   },
   "dependencies": {
     "@reef-knot/connect-wallet-modal": "0.3.0",
-    "@reef-knot/web3-react": "0.2.0"
+    "@reef-knot/web3-react": "0.2.0",
```

Comment:
Shouldn't these include patches? e.g. `^0.2.0`

Reply:
Every patch version of `reef-knot/web3-react` will bump `reef-knot` too, this is managed by `changesets` tool. I prefer strict versions in this case.

Reply:
makes sense